app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,8 @@
+<%provide :title, t(".provide_title")%>
+	<h1><%=t".title"%></h1>
+	<p>
+		<%= t ".service_html_first"%>
```

Comment:
check lại toàn bộ view nhé
sai hết indent
tab 2 space thôi

Reply:
ơ s ở vs code em nó tab 2 space mà chị nhỉ s push lên đây nó lại bị 4 space thế ạ

Reply:
![image](https://user-images.githubusercontent.com/72131384/168210190-5888a2a8-26b2-4d95-8ee9-7e9a65e33430.png)
Em đổi space thành thế này nhé